connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -656,6 +567,19 @@ public KafkaProducer<byte[], byte[]> createProducer(Map<String, Object> producer
         return producer;
     }
 
+    private void addDefaultBrokerPropsIfAbsent(Properties brokerConfig, int numBrokers) {
+        putIfAbsent(brokerConfig, KafkaConfig.DeleteTopicEnableProp(), "true");
+        putIfAbsent(brokerConfig, KafkaConfig.GroupInitialRebalanceDelayMsProp(), "0");
+        putIfAbsent(brokerConfig, KafkaConfig.OffsetsTopicReplicationFactorProp(), String.valueOf(numBrokers));
+        putIfAbsent(brokerConfig, KafkaConfig.AutoCreateTopicsEnableProp(), "false");
+    }
+
+    private static void putIfAbsent(final Properties props, final String propertyKey, final Object propertyValue) {
+        if (!props.containsKey(propertyKey)) {
+            props.put(propertyKey, propertyValue);
+        }
+    }
```

Comment:
Can't we just use `Properties::putIfAbsent`? IIRC the only reason we have the variant below for `Map<String, Object>` is because it was implemented when we still supported Java versions before 1.8.

```suggestion
    private void addDefaultBrokerPropsIfAbsent(Properties brokerConfig, int numBrokers) {
        brokerConfig.putIfAbsent(KafkaConfig.DeleteTopicEnableProp(), "true");
        brokerConfig.putIfAbsent(KafkaConfig.GroupInitialRebalanceDelayMsProp(), "0");
        brokerConfig.putIfAbsent(KafkaConfig.OffsetsTopicReplicationFactorProp(), String.valueOf(numBrokers));
        brokerConfig.putIfAbsent(KafkaConfig.AutoCreateTopicsEnableProp(), "false");
    }
```

Reply:
Ah whoops, my bad! I can go ahead and clean up the map variant in this PR as well if you'd like?

Reply:
Either way is fine, not a blocker but would be nice 👍